boards/common/nrf52/Makefile.features

```diff
@@ -9,3 +9,8 @@ endif
 
 # Various other features (if any)
 FEATURES_PROVIDED += ble_nimble
+
+ifneq (,$(findstring nrf52832, $(CPU_MODEL)))
+  # include common nrf52832 features
+  include $(RIOTBOARD)/common/nrf52/nrf52832/Makefile.features
```

Comment:
now look at that included file, we might simply want to drop `nrf52832/Makefile.features` and decleare the softdevice feature in the if block directly. If ever (and that is to be quite unexpected) there is additional feature(s), we can always re-introduce that file...

Reply:
Done, squashed and rebased (there was a conflict introduced by #12290)